models/comments.model.js

```diff
@@ -22,3 +22,16 @@ exports.addCommentByArticleId = async (comment, article_id) => {
   );
   return rows[0];
 };
+
+exports.removeCommentById = async (comment_id) => {
+  if (!parseInt(comment_id)) {
+    return Promise.reject({ status: 400, msg: "comment_id is not a number" });
+  }
+  const { rows } = await db.query(`SELECT comment_id FROM comments;`);
+  const validCommentIds = rows.map((id) => id.comment_id.toString());
+  if (!validCommentIds.includes(comment_id)) {
+    return Promise.reject({ status: 404, msg: "comment_id does not exist" });
+  } else {
```

Comment:
All of this is SQL stuff. Imagine Facebook pulling down every comment ever onto your local machine to check if a comment exists.